src/WebJobs.Script/Workers/Http/HttpWorkerRoute.cs

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.Azure.WebJobs.Extensions.Http;
+
+namespace Microsoft.Azure.WebJobs.Script.Workers.Http
+{
+    /// <summary>
+    /// Route mapping for a custom handler HTTP worker.
+    /// </summary>
+    public sealed class HttpWorkerRoute
+    {
+        /// <summary>
+        /// Gets or sets route template.
+        /// </summary>
+        public string Route { get; set; }
+
+        /// <summary>
+        /// Gets or sets the authorization level.
+        /// </summary>
+        public AuthorizationLevel? AuthorizationLevel { get; set; }
```

Comment:
Thinking we want this property and it's host.json name to match our standard http trigger "authLevel" naming? https://github.com/Azure/azure-webjobs-sdk-extensions/blob/b8d5f24bb827ddb4e43410cd713cae3cbafc8bfb/src/WebJobs.Extensions.Http/HttpTriggerAttribute.cs#L69

Why have it differ?